lib/sqlalchemy/sql/schema.py

```diff
@@ -1448,6 +1461,7 @@ def __init__(
         quote: Optional[bool] = None,
         system: bool = False,
         comment: Optional[str] = None,
+        system_versioning: bool = None,
```

Comment:
this would be `Optional[]`, but I was hoping we didn't need to have "system_versioning" to be a parameter / public attribute of Column.   we want there to be just one API way to do this, makes it easier to document, test and maintain.   However you have one test that is using "SYSTEM VERSIONING" on a column without anything on the table, so I need to understand that use case.

also note that to render GENERATED AS etc. we should be using the [Computed](https://docs.sqlalchemy.org/en/14/core/defaults.html?highlight=computed#computed-ddl) object for that, if we need to add more parameters to `Computed` to support this, that's where it would go.

Reply:
Good catch on optional. I added this to add the column-specific `WITH SYSTEM VERSIONING` / `WITHOUT SYSTEM VERSIONING` that can opt an individual column in/out. Example from maria:

```sql
CREATE TABLE t (
   x INT,
   y INT WITHOUT SYSTEM VERSIONING
) WITH SYSTEM VERSIONING
```

I don't think this is actually in SQL:2011 but I at least request PG to implement the same [on their chain discussing SV](https://www.postgresql.org/message-id/flat/CALAY4q-cXCD0r4OybD=w7Hr7F026ZUY6=LMsVPUe6yw_PJpTKQ@mail.gmail.com)